### PR TITLE
Add Swallow

### PR DESCRIPTION
**Pull request by BenWoodford76**
Want build preview for now, can be merged post final show

### CODE REVIEW

File: _shows/17_18/swallow.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title: Swallow
+playwright: Stef Smith
+period: Spring
+season: In House
+season_sort: 190
+date_start: 2018-02-28
+date_end: 2018-03-02
```
Comment (nhyty) [inline _shows/17_18/swallow.md]: Can we have the correct end date please

**Comment by nhyty** [comment]
@BenWoodford76 can you fix it first though pls

**Comment by BenWoodford76** [comment]
Sorted

File: _shows/17_18/swallow.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  - type: poster
+    image: C7QjcF4
+  - type: poster
+    image: 2DPSVR4
+  - type: poster
+    image: 7dWMfd6
+  - type: trailer
+    video: CGRCrZg
+    title: Trailer
+    type: backstage
```
Comment (samozzy) [inline _shows/17_18/swallow.md]: Is there a missing `-` here?

**Comment by nhyty** [comment]
Yes

**Comment by BenWoodford76** [comment]
@nhyty Fixed this

**Pull request merged**